documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -1,3 +1,17 @@
+= Upgrading from 10.1 to 11.0
```

Comment:
So technically this isn't just 11.0 still, which is why I was asking on Zulip if we should put something here. Cause technically this change will be in 10.1.1 as well.